master/static/migrations/20220215164216_add-cluser-id-cluser-heartbeat.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE public.cluster_id ADD COLUMN cluster_heartbeat timestamp not null DEFAULT (DATE_TRUNC('millisecond', now() at time zone 'utc')::timestamp);
```

Comment:
can you add new line endings to the sql files and format them a bit?

Reply:
done